src/main.js

```diff
@@ -64,12 +65,23 @@ function processAppArgs(args) {
   }
 }
 
-function rotateUsers() {
-  const updatedUsers = userService.rotate()
-  const activeUserCount = updatedUsers.filter(u => u.active).length
-  if (activeUserCount > 1) {
-    const label = getNotificationLabel(activeUserCount, true)
-    notificationService.showCurrentAuthors({ title: `${label} rotated to:` })
-    mb.window.webContents.send(CHANNELS.USERS_UPDATED, updatedUsers)
+function startUp() {
+  const isSecondInstance = !mb.app.requestSingleInstanceLock()
+  if (isSecondInstance) {
+    mb.app.exit()
+    return
   }
+
+  mb.on('ready', handleAppReady)
+  mb.on('after-create-window', handleAfterCreateWindow)
+  mb.app.on('second-instance', (_, argv) => {
+    processAppArgs(argv)
+  })
+
+  processAppArgs(process.argv)
+
+  const appExecutablePath = mb.app.getPath('exe')
+  install(process.platform, appExecutablePath)
 }
+
+startUp()
```

Comment:
If Im reading the changes in this file correctly, the main changes are:
- reordering functions
- menubar had some of its initialization arguments moved into a `browserWindow` key.

Am I leaving anything out?

Reply:
That's basically the gist of it. One other notable change is that the electron single instance api changed a little. Instead of `makeSingleInstance()`, it is now `requestSingleInstanceLock()` and they added a new `second-instance` event that we listen to to process the args on.